src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java

```diff
@@ -111,17 +113,17 @@ public boolean isAdVisible(final WebElement element, final String selector,
       wait.until(new ExpectedCondition<Object>() {
         @Override
         public Object apply(WebDriver driver) {
-          File adImg = shooter.captureWebElement(element, driver);
-          PageObjectLogging.log("ScreenshotsComparison", "Ad image in " + selector, true, driver);
-          boolean areFilesTheSame = imageComparison.areFilesTheSame(backgroundImg, adImg);
-          adImg.delete();
-          return !areFilesTheSame;
+          BufferedImage adImg = shooter.takeScreenshot(element, driver);
```

Comment:
I strongly recommend you to move calculations to another class, this class is a helper for images on wiki page, not a place to store every image related staff